draft-schwartz-ds-glue.md

```diff
@@ -192,39 +198,38 @@ example    3600 IN NS    ns1.example
 ns1.example 600 IN AAAA  2001:db8::1
 ~~~
 
-A zone in this configuration can optionally use an NSEC DSGLUE record to indicate that there is no IPv4 address:
+A zone in this configuration can optionally use an empty DSGLUE record to indicate that there is no IPv4 address:
 
 ~~~
 $ORIGIN com.
-example 600 IN DS $DSGLUE(., NS, ns1.example.com.)
-            IN DS $DSGLUE(ns1., AAAA, 2001:db8::1)
-            IN DS $DSGLUE(ns1., NSEC, ns1.example.com. AAAA)
+example 600 IN DS $DSGLUE(., NS, 3600, [ns1.example.com.])
+            IN DS $DSGLUE(ns1., AAAA, 600, [2001:db8::1])
+            IN DS $DSGLUE(ns1., A, 7200, [])
 ~~~
 
-This arrangement prevents an adversary from inserting forged records for ns1.example.com or _dns.ns1.example.com into the delegation response.
+This arrangement prevents an adversary from inserting forged A records for ns1.example.com into the delegation response.
 
-Note that although there is an NSEC record denying the existence of A records for ns1.example.com, it is treated as a glue record that only applies during delegation, so such records can still be resolved if they exist.
+Note that this negative answer is treated as glue that only applies during delegation, so A records for ns1.example.com can still be resolved if they exist.
```

Comment:
If we are getting trusted glue in the delegation response, what is the incentive to re-resolve the in-bailiwick NS names? Any secure resolution of the NS name could only happen over the DoT channel established after processing the DSGLUE RRs.